pkgs/development/tools/nwjs-ffmpeg-prebuilt/default.nix

```diff
@@ -0,0 +1,51 @@
+{ fetchurl
+, unzip
+, lib
+, tree
+, stdenv
+}:
+
+let
+  bits = if stdenv.hostPlatform.system == "x86_64-linux" then "x64" else "ia32";
+  version = "0.87.0";
+in
+stdenv.mkDerivation {
+  pname = "nwjs-ffmpeg-prebuilt";
+  inherit version;
+
+  src =
+    let
+      hashes = {
+        "x64" = "0ala43qhg0zi662d3yahbcxrnpx9439jlqwdsi58ck5p36ih9mp0";
+        "ia32" = "0ala43qhg0zi662d3yahbcxrnpx9439jlqwdsi58ck5p36ih9mp0";
```

Comment:
were these hashes checked properly? these are the same hash.
Also please use SRI hash format.